release_notes/ocp-4-16-release-notes.adoc

```diff
@@ -845,11 +845,11 @@ This feature provides two new APIs, `AdminNetworkPolicy` (ANP) and `BaselineAdmi
 For more information, see xref:../networking/ovn_kubernetes_network_provider/converting-to-dual-stack.adoc#nw-dual-stack-convert_converting-to-dual-stack[Converting to a dual-stack cluster network] in _Converting to IPv4/IPv6 dual-stack networking_.
 
 [id="ocp-4-16-networking-sdn-ovnk-live-migration_{context}"]
-==== Live migration to the OVN-Kubernetes network plugin
+==== Limited live migration to the OVN-Kubernetes network plugin
 
 Previously, when migrating from OpenShift SDN to OVN-Kubernetes, the only available option was an _offline_ migration method. This process included some downtime, during which clusters were unreachable.
```

Comment:
Should we include a link to `an _offline_ migration method`?

Reply:
I _think_ because this offline is just one or two modules above the link to the live and this is specifically calling out the live we are ok @weliang1